addons/external_intercom/baseHeadgear.hpp

```diff
@@ -0,0 +1,20 @@
+private _baseHeadgear = [
```

Comment:
This should be a config property, what should mods do to add their supported helmets to this?
I'd say make it config property, and scan once at preStart and save it in uiNamespace.

Or alternatively instead of scanning, could also make a global config class/array that mods can append to